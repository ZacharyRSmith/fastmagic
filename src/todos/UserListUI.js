import React from 'react';

const UserListUI = ({ users }) => {
  return (
    <ul>
      {users.map(user =>
        <li key={user.id}>
          {user.name}
        </li>
      )}
    </ul>
  );
};

export default UserListUI;
