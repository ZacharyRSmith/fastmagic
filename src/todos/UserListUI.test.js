import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import UserListUI from './UserListUI';

it('renders without crashing', () => {
  const wrapper = shallow(
    <UserListUI {...{
      users: [{ id: '1', name: 'Spot' }]
    }}/>
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});
