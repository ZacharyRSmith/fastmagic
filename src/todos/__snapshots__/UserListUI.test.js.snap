exports[`test renders without crashing 1`] = `
<ul>
  <li>
    Spot
  </li>
</ul>
`;
